engine/rowset/src/main/java/io/deephaven/engine/rowset/RowSet.java

```diff
@@ -170,7 +170,7 @@ interface RangeIterator extends SafeCloseable {
         /**
          * <p>
          * Advance the current iterator position until {@code currentRangeStart()} and {@code currentRangeEnd()} are
-         * both greater than or equal to ‘v’. This may or may not move the iterator to the next range: if ‘v’ is inside
+         * both greater than or equal to `v`. This may or may not move the iterator to the next range: if `v` is inside
```

Comment:
Does markdown syntax work in html? Maybe you want `{@code}` or `<code></code>` tags?